--- a/feeds/kenzok8/tun2socks/Makefile
+++ b/feeds/kenzok8/tun2socks/Makefile
@@ -4,9 +4,9 @@ PKG_NAME:=tun2socks
 PKG_VERSION:=2.6.0
-PKG_RELEASE:=61
+PKG_RELEASE:=60
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/xjasonlyu/tun2socks.git
-PKG_SOURCE_VERSION:=c36137df92a9f9a698a1bae73518c445802df82d
+PKG_SOURCE_VERSION:=9c2ce399799536655a1a1cb59561e270be767b87
 
 PKG_MAINTAINER:=Konstantine Shevlakov <shevlako@132lan.ru>
